v3/src/components/graph/plots/scatter-plot/scatter-plot.tsx

```diff
@@ -138,9 +141,12 @@ export const ScatterPlot = observer(function ScatterPlot({ pixiPoints }: IPlotPr
             [xAttrID, secondaryAttrIDsRef.current[plotNumRef.current]])
       }
     }
-  }, [layout, dataConfiguration, dataset, dragID])
+  }, [layout, dataConfiguration, dataset, dragID, numExtraPrimaryBands, numExtraSecondaryBands])
 
   const onDragEnd = useCallback(() => {
+    // We turn these off first so that transitions will work properly when we update the data values
+    dataset?.endCaching()
+    appState.endPerformance()
```

Comment:
I trust that you considered the prior comment that justified moving these calls later?